### PR TITLE
Implement ELB installation

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

**Comment by tgalopin** [comment]
I'm going to review this asap, thanks for the feature!

File: res/acmephp.conf.dist [hunk 1]
```diff
@@ -57,6 +57,25 @@ storage:
...
         #   # timeout: 30
 
+        #
+        # Install the certificate in an AWS ELB (Classic Load Balancer)
+        #
+        # - action: install_aws_elb
+        #   region: eu-west-1
+        #   loadbalancer: my_elb
+        #   # certificate_prefix: acmephp_
+        #   # cleanup: true
```
Comment (tgalopin) [inline res/acmephp.conf.dist]: Could we use `cleanup_old_certificate` instead, to be clearer? Not a blocker for me though.

File: src/Cli/Action/AwsClientFactory.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AcmePhp\Cli\Action;
```
Comment (tgalopin) [inline src/Cli/Action/AwsClientFactory.php]: Perhaps a dedicated `Aws` namespace would be more appropriate?

Comment (jderusse) [reply src/Cli/Action/AwsClientFactory.php]: :+1:

**Pull request merged**